styles.css

```
@@ -4781,18 +4781,16 @@ input:checked + .slider::before {
+.volume-bar, .fs-volume-bar {
     position: relative;
 }
-
 .volume-tooltip {
     position: absolute;
     bottom: calc(100% + 8px);
     left: 50%;
     transform: translateX(-50%);
     padding: 4px 8px;
-    background: rgb(20, 20, 20, 0.9);
+    background: rgba(20, 20, 20, 0.9);
```

Stylelint (17.14.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)